var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -95,6 +95,9 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
     variant('nvptx',
             default=False,
             description='Target nvptx offloading to NVIDIA GPUs')
+    variant('enable_bootstrap',
```

Comment:
```suggestion
    variant('bootstrap',
```
I think this should be called just `bootstrap`, since the `enable` is redundant.